### PR TITLE
[CyclingUK] Added Front-end

### PR DESCRIPTION
**Pull request by lucascumsille**
Fixes part of: https://github.com/mysociety/societyworks/issues/3994

Not for this PR, but I added a couple of TODO that can be found using this term "TODO CUK"

### Front page

Front page

### Map page

map-1
map-2
map-3

### Confirmation
Not found
Update
Thank you

### Local Alerts
Local alerts

### Footer

Footer

[skip changelog]

### CODE REVIEW

File: web/cobrands/cyclinguk/images/cards/card-2.webp
Comment (dracos) [inline web/cobrands/cyclinguk/images/cards/card-2.webp:1]: The three card pics add c. 1.25MB to any page they're shown on. Along with the 2.45MB other picture as well this makes these pages significantly heavier. [edit: 2.45MB image has been shrunk, can these ones be as well?]

File: templates/web/cyclinguk/email_sent.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            [% loc("Can&rsquo;t find our email? Check your spam folder&nbsp;&ndash; that&rsquo;s the solution 99% of the time.") %]
+        </p>
+    
+    </div>
+</div>
+[% END %]
+
+[% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title = loc('Create a report') %]
+
+[% TRY %][% INCLUDE 'emails/_emails_content.html' %][% CATCH file %][% END %]
```
Comment (dracos) [inline templates/web/cyclinguk/email_sent.html:29]: We deliberately do not include anything else on the check your email page so as not to distract people when the thing they *have* to do now is check their email and confirm their thing. So we can add these things here, but I'm not sure we should. Looking at the page, I just completely gloss over the important message of checking your email, assuming it's some kind of banner message I can ignore
image

File: web/cobrands/cyclinguk/images/cyclinguk-hero.webp
[the remark this answers is not in the document]
Comment (dracos) [reply web/cobrands/cyclinguk/images/cyclinguk-hero.webp:1]: I see it's now been shrunk, thanks, but still is same picture as other one, so they should use the same (small) image.

File: templates/web/cyclinguk/tokens/confirm_problem.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+[% INCLUDE
+  next_steps.html,
+  just_interacted = 1,
+  utm_content = 'problem confirmed',
+  share_url = c.cobrand.base_url_for_report(report) _ report.url,
+  twitter_comment = loc('I just reported a problem on @fixmystreet')
+%]
+
+[% TRY %][% INCLUDE 'emails/_emails_content.html' %][% CATCH file %][% END %]
```
Comment (dracos) [inline templates/web/cyclinguk/tokens/confirm_problem.html:34]: Any reason you didn't put the contents of _emails_content.html in next_steps.html instead?

File: templates/web/cyclinguk/tokens/confirm_problem.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+[% pre_container_extra = BLOCK %]
```
Comment (dracos) [inline templates/web/cyclinguk/tokens/confirm_problem.html:1]: I can't really see a nicer way of doing this at present that doesn't involve littering the base templates with 'IF cobrand==cyclinguk', or moving all the "main content" parts out into separate templates that can be included, which isn't exactly ideal either. SO perhaps leave as is for now (though I think reusing next_steps unless there's a reason not to do that) and we can see if we can think of something better later.

**Review by dracos — CHANGES_REQUESTED** [review]
Hope that all makes sense, let me know if not!

File: web/cobrands/cyclinguk/images/cyclinguk-hero.webp
[the remark this answers is not in the document]
Comment (lucascumsille) [reply web/cobrands/cyclinguk/images/cyclinguk-hero.webp:1]: At the moment the CTA one is a placeholder. Waiting on the client to provide the real one.

File: templates/web/cyclinguk/tokens/confirm_problem.html (hunk 2 above)
Comment (lucascumsille) [reply templates/web/cyclinguk/tokens/confirm_problem.html:34]: No reason at all, I didn't realise that one. I'm pushing a commit that should solve that.

File: templates/web/cyclinguk/email_sent.html (hunk 1 above)
Comment (lucascumsille) [reply templates/web/cyclinguk/email_sent.html]: Sounds good and it makes sense. So What I've done now is display the content below on:

- 404
- Confirm Alert
- Confirm Problem
- Confirm Update

All the errors or anything that requires the user to execute an action shouldn't have that. I might need to add something to those ones, so the hero and footer doesn't clash.

File: web/cobrands/cyclinguk/images/cards/card-2.webp
Comment (lucascumsille) [reply web/cobrands/cyclinguk/images/cards/card-2.webp:1]: No probs I optimised those assets. Now the three of them in total is 189kb.

File: web/cobrands/cyclinguk/layout.scss [hunk 4]
```diff
@@ -1,9 +1,229 @@
...
 @import "../sass/layout";
 
-body.mappage #site-logo {
-    padding: 0;
-    height: 60px;
-    width: 265px;
-    background-size: 234px 53px;
-}
\ No newline at end of file
+.hidden-desktop {
```
[the remark this answers is not in the document]
Comment (lucascumsille) [reply web/cobrands/cyclinguk/layout.scss]: Thanks I got rid of that one.

File: templates/web/cyclinguk/tokens/confirm_problem.html (hunk 3 above)
Comment (lucascumsille) [reply templates/web/cyclinguk/tokens/confirm_problem.html:1]: I'll leave as it is for now. I'm using currently using `next_steps` and got rid of the file I created. =)

**Review by dracos — APPROVED** [review]

**Pull request merged**